Projects/views.py

```diff
@@ -354,3 +334,43 @@ def change_label(request):
         issue.save()
         return HttpResponse(res)
     else: return HttpResponseBadRequest()
+
+def new_issue(request):
+    print('adding issue')
+    if request.method == 'POST':
+        if request.user.is_staff:
+            print('in add issue method')
+            mentor = request.user
+            title_issue = request.POST.get('title_issue')
+            link_issue = request.POST.get('link_issue')
+            title_project = request.POST.get('title_project')
+            link_project = request.POST.get('link_project')
+            level = int(request.POST.get('level'))
```

Comment:
here you should put try and catch to handle char input as level and points, because int('char') will raise exception

Reply:
but sir user cannot enter text data i.e. alphabets on form so here it won't create problem.

Reply:
again a char can be sent using console :smile: